src/utils/browser.js

```diff
@@ -4,10 +4,11 @@ export default {
         return /Android|webOS|iPhone|iPod|BlackBerry/i.test(navigator.userAgent)
     },
     numberFormatter(num) {
```

Comment:
I suggest add `num = Number(num)` here. Then change all `formatter(Number(x))` to `formatter(x)`.